packages/chain-bridge/src/libraries/BridgeTypes.sol

```diff
@@ -21,6 +21,9 @@ library BridgeTypes {
         uint32 calldataSize; // Size of expected return calldata (for read operations)
         uint128 msgValue; // Native value to forward (for operations requiring value)
         bytes options; // Additional adapter-specific parameters
+        // --- Protocol token fee support (optional) ---
+        bool payInProtocolToken; // If true, pay fees in protocol token when supported
```

Comment:
I'm not sure the `feeToken` is required here ( or used) if the token is defined in side of the adapter. Unless that serves another purpose